crispycuts/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
+
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById('root')
+// );
+
+// // If you want to start measuring performance in your app, pass a function
+// // to log results (for example: reportWebVitals(console.log))
+// // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+// reportWebVitals();
+
+const Body= ()=>{
+  return(
+    <div id="body">
+      <Header />
+      <Card 
+        className='section'
+        img='./Capture1.PNG'
+        title='About the Company'
+        description='Lorem ipsum dolor sit amet...'
+      
+      />
+      <ContactContainer/> 
+    </div>
+  )
+}
+
+const Header = () =>{
+  return(
+    <div className='header'>
+      <span className='header-title'>
+        Company
+      </span>
+      <br/>
+      <span className='header-text'>
+        We specialize in something...
+      </span>
+    </div>
+  )
+}
+
+const Card = (props) => {
```

Comment:
With typescript, you'll need to define the type for the props. You can either define the type outside the the functional component or inside, up to you. But if the type is quite big, it makes sense to define it outside so your code doesn't get too sloppy. Since your props have `img` which is a `string`, `title` which is a `string` and description which is a string, your type will be `{img: string; title: string; description: string}`. 

To define outside of the function:

`type CardProps = {img: string; title: string; description: string}`
`const Card = (props: CardProps) => {.....`

To define inside of the function: 
`const Card = (props: {img: string; title: string; description: string}) => {.....`